staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/BUILD

```diff
@@ -36,6 +36,8 @@ go_library(
         "//vendor/github.com/coreos/etcd/client:go_default_library",
         "//vendor/github.com/coreos/etcd/clientv3:go_default_library",
         "//vendor/github.com/coreos/etcd/pkg/transport:go_default_library",
+        "//vendor/github.com/grpc-ecosystem/go-grpc-prometheus:go_default_library",
```

Comment:
To resolve this merge conflict, you should be able to just revert this file (`staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/BUILD`) back to the upstream version, then re-run `hack/update-bazel.sh`, which should re-add these lines.

Trying to resolve the conflict manually is doable but painful, since so many (but not all) of these `//vendor/k8s.io` dependencies turned into `//staging/src/k8s.io`.